build.sbt

```diff
@@ -89,15 +89,15 @@ inThisBuild(
     testFrameworks := List(new TestFramework("munit.Framework")),
     resolvers += Resolver.sonatypeRepo("public"),
     // faster publishLocal:
-    publishArtifact.in(packageDoc) := "true" == System.getenv("CI"),
-    publishArtifact.in(packageSrc) := "true" == System.getenv("CI"),
+    (packageDoc / publishArtifact) := "true" == System.getenv("CI"),
+    (packageSrc / publishArtifact) := "true" == System.getenv("CI"),
     turbo := true,
     useSuperShell := false // overlaps with MUnit test failure reports.
   )
 )
 
 name := "mdocRoot"
-skip in publish := true
+(publish / skip) := true
```

Comment:
I think in those positions it should work without the parentheses?

Reply:
Ha, @gabro literally asked the same thing in Metals. Yea, it will. I did this with [this Scalafix rule](https://eed3si9n.com/syntactic-scalafix-rule-for-unified-slash-syntax), and it seems to default to this. My assumption is that this is does so that in the situation that:

```scala

a / b := true
```

needs to become:

```scala
(a / b) / c := true
```

That it can easily be done without any confusion and avoid someone accidentally doing `(a / b / c)`. Again, I'm not fully sure.

Reply:
it depends on the associativity of `/`, but if it's left associative (as I think it should be), then `(a / b) / c` should be the same as `a / b / c`.

Reply:
Huh, how have I not realised this before... Of course these would all be the same

```scala
((Compile / doc) / scalacOptions)

```scala
(Compile / doc) / scalacOptions
```

```scala
Compile / doc / scalacOptions
```

🤔 so I guess I'm not fully sure why it wraps everything in `()`. Do you all have a preference on style here then?

Reply:
I guess it wraps in `()` for cases like `stuff.in(hello).value` not being converted to `stuff / hello.value` (which wouldn't compile)